src/Recurr/Transformer/TextTransformer.php

```diff
@@ -14,7 +14,7 @@ public function __construct(TranslatorInterface $translator = null)
         $this->translator = $translator ?: new Translator('en');
     }
 
-    public function transform(Rule $rule)
+    public function transform(Rule $rule, $exclude = array())
```

Comment:
```suggestion
    public function transform(Rule $rule, $exclude = [])
```
https://github.com/simshaun/recurr/blob/master/composer.json#L16

Old PHP 5.3 compat is not needed :) (https://github.com/wdes/coding-standard#rules-to-disable-for-php--54-compatibility)